src/Microsoft.Sbom.Api/Config/SbomToolCmdRunner.cs

```diff
@@ -131,5 +139,23 @@ public async Task Generate(GenerationArgs generationArgs)
                 IsFailed = true;
             }
         }
+
+        /// <summary>
+        /// Prints the version of the tool.
+        /// </summary>
+        [ArgActionMethod]
+        [ArgShortcut("--version")]
```

Comment:
if we are following the --version convention, we can add another shortcut, `-v`

Reply:
I considered adding it but saw that `-v` is used as a verbosity parameter inside the generate and validate actions. Adding it shouldn't cause any issues but thought it might be confusing for users.